### DISCUSSION TITLE
Access enabled, but "Extension activation failed: "User not authorized""

### DISCUSSION THREAD

**Discussion by Looouuu** [community/community#9697] [Copilot Conversations] [answered]
Hello !

As the title says, when I login on VSCode, I get

[INFO] [default] [2022-01-06T12:50:33.949Z] Experimentation service initialized
[INFO] [default] [2022-01-06T12:50:33.955Z] Stringified features: {}
[INFO] [default] [2022-01-06T12:50:33.955Z] Feature for copilotaa: false
[INFO] [default] [2022-01-06T12:50:33.955Z] Feature for copilotincludeprompt: none
[INFO] [default] [2022-01-06T12:50:33.955Z] Feature for copilotset1: undefined
[INFO] [default] [2022-01-06T12:50:33.955Z] Feature for copilotghgh: false
[INFO] [default] [2022-01-06T12:50:33.955Z] Feature for copilotfim: false
[INFO] [auth] [2022-01-06T12:50:34.271Z] Invalid copilot token: missing token: 404 
[ERROR] [default] [2022-01-06T12:50:34.273Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

But when I go on https://github.com/features/copilot/signup I get "GitHub Copilot is already enabled on your account! Install GitHub Copilot now." and I used Copilot in the past, it worked great.

Any idea ? 

Thanks :)

**Comment by Looouuu** [comment, marked as answer]
After removing vscodium and installing vscode, it works. Weird !

**Comment by ishivamshukla** [comment]
You're still on the waitlist. 
and wait we should use it after receiving the official mail.
